.github/ISSUE_TEMPLATE/1_general_bug_report.yaml

```diff
@@ -0,0 +1,101 @@
+name: Report a bug
+description: Errors and regression reports with complete reproducing test cases and/or stack traces.
+labels: ["status:needs-triage", "bug"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Many bug reports are often **USAGE QUESTIONS, NOT BUGS**. If you do not have a strong understanding 
```

Comment:
```suggestion
        Bug reports are often **usage questions, not bugs**. If you do not have a strong understanding 
```

Reply:
It kind of felt like this was yelling at me.

Reply:
I'm down. I removed a bunch of other caps text in this PR.